### PR TITLE
Issue113 Fixes

### PR DESCRIPTION
**Pull request by kb1lqc**
This PR addresses the symptoms in #113 
- Changed node symbole to an actual "node" with the "n" APRS symbol
- Added string to print how many stations are being logged for each update
- Removed printing of GPS Fix warning and any unnecessary data
- Added a logger.info() for each APRS telemetry packet function. This is for identification of future improvements. It's not necessary right now but will help make program more stable and recover from errors gracefully.
- Added programmingerror exception to SQL insertion function. This helps fail gracefully with bad data input
- Added rollback() to exception to allow database to work after exception.
- Fixed up some comment wording
- Fixed checking for callsign="%" in query DB fuction which caused a LIMIT=1 SQL query to be tacked onto all queries. This caused Issue #113.
  - If callsign is specified then the callsign is searched and returned stations are only nodes with that callsign. When no callsign is specified the QSL query is kept generic.

This code has a few known deficiencies
- Does not gracefully handle network errors and try to reconnect #119
- Telemetry will cause an error if no stations are returned. This is both a deficiency of APRS for not handling it well and telemetry for also not handling zero station /station API calls well #120.

### CODE REVIEW

**Comment by kb1lqc** [comment]
OK @kb1lqd this is ready for review

File: Applications/APRS/aprs.ini [hunk 1]
```diff
@@ -16,7 +16,7 @@ DATATYPEIDENT=!
 DESTADDRESS=GPSFDY
 SYMBOLTABLE=/
 ALTSYMBOLTABLE=\
-SYMBOL=N
+SYMBOL=n
```
Comment (kb1lqd) [inline Applications/APRS/aprs.ini:19]: This is good, I was wondering why we were showing up as NTS stations...

File: Applications/Telemetry/telemetry.py [hunk 2]
```diff
@@ -587,16 +595,20 @@ def queryStationsDb(parameters):
     sqlEnd = "GROUP BY SOURCECALLSIGN, SOURCEID ORDER BY EPOCH DESC"
 
     # detect if callsign/nodeid provided, return the last time it was heard
-    # Since a callsign was specified, simply search the entire db for it
-    # and return the last epoch time it was heard.
-    timeTuple = (0, time.time())
+    if parameters["CALLSIGN"] != "%":
```
Comment (kb1lqd) [inline Applications/Telemetry/telemetry.py:598]: Seems to work fine. APRS is working with multiple stations. 1 Local and 1 Remote RF.

File: Applications/APRS/aprs.ini (hunk 1 above)
Comment (kb1lqd) [reply Applications/APRS/aprs.ini:19]: Verified that intended node type is shown on APRS.fi

**Review by kb1lqd — APPROVED** [review]
Reviewed code = Looks good
Ran APRS with units = All units local and remote verified on APRS.fi.

**Pull request merged**